src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts

```diff
@@ -164,14 +164,14 @@ export class ProfileDashboardComponent implements OnInit, OnDestroy {
     return [...habitsOnView, ...allHabits.slice(currentNumberOfHabitsOnView, currentNumberOfHabitsOnView + this.numberOfHabitsOnView)];
   }
 
-  private sortHabitsAsc(habitsArray: HabitAssignInterface[]): Array<HabitAssignInterface> {
+  private sortHabitsData(habitsArray: HabitAssignInterface[]): Array<HabitAssignInterface> {
     return habitsArray.sort((firstHabit, secondHabit) => {
-      if (firstHabit.habit.id > secondHabit.habit.id) {
-        return 1;
-      }
-      if (secondHabit.habit.id > firstHabit.habit.id) {
+      if (firstHabit.createDateTime > secondHabit.createDateTime) {
         return -1;
       }
+      if (secondHabit.createDateTime > firstHabit.createDateTime) {
+        return 1;
+      }
       return 0;
     });
```

Comment:
I would simplify it:
return firstHabit.createDateTime > secondHabit.createDateTime ? -1 : 1

Reply:
Thank you. Already fixed